spec/system/admin/products_v3/products_spec.rb

```diff
@@ -157,6 +157,36 @@
     end
   end
 
+  describe "Actions columns (edit)" do
+    let!(:variant_a1) {
+      create(:variant,
+             product: product_a,
+             display_name: "Medium box",
+             sku: "APL-01",
+             price: 5.25)
+    }
+    let!(:product_a) { create(:simple_product, name: "Apples", sku: "APL-00") }
+
+    before do
+      visit admin_products_v3_index_url
+    end
+
+    it "shows an actions memu with an edit link when clicking on icon for product" do
+      within row_containing_name("Apples") do
+        page.find(".vertical-ellipsis-menu").click
+        expect(page).to have_link "Edit", href: spree.edit_admin_product_path(product_a)
```

Comment:
I was thinking perhaps the test should actually click on the link to verify that it works. I guess it's possible that some random Javascript could `preventDefault` and prevent this link from working. 
But this seems sufficient, and much more efficient 👍